src/main/java/ca/bc/gov/api/oracle/legacy/dto/ClientPublicViewDto.java

```diff
@@ -0,0 +1,15 @@
+package ca.bc.gov.api.oracle.legacy.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
```

Comment:
You can remove lombok annotations

Reply:
Removed lombok annotations but left @Data, should it be removed as well?